src/components/LanguageSelector.jsx

```diff
@@ -41,7 +42,7 @@ class LanguageSelector extends Component {
       existingLanguages.push(languageOption);
       menuLanguages = menuLanguages.filter(option => option !== languageOption);
     });
-    
+
```

Comment:
I guess it could perhaps return `languageOption`. Sorry, I'm not getting what the return value should be.